tests/test_multieurlex_utils.py

```diff
@@ -0,0 +1,102 @@
+import os
+from unittest.mock import patch
+
+import datasets
+import pyarrow as pa
+from datasets.formatting import PythonFormatter
+from datasets.formatting.formatting import LazyRow
+
+from arc_spice.data import multieurlex_utils
+
+# def extract_articles(
```

Comment:
to remove, sorry

Reply:
The whole file? or the `extract_articles'?

Reply:
just the commented out sig